project/build.properties

```diff
@@ -1 +1,2 @@
 sbt.version=1.7.3
+
```

Comment:
can remove the space

Reply:
It's actually best practice to leave a new line at the end of source files. Intellij automatically adds them for me